spock-specs/src/test/groovy/org/spockframework/smoke/extension/StepwiseSpecs.groovy

```diff
@@ -78,18 +84,25 @@ class Foo extends Specification {
   def step1() { expect: true }
   def step2() { expect: true }
   def step3() { expect: true }
+  def step4() { expect: true }
```

Comment:
Adding 

```java
  def setup () {
    println specificationContext.currentFeature.name
  }
```

will print for `step3`

```
step1
//
step2
step1
//
step3
step2
step1
```

Reply:
Not sure what you mean. For `step3` it is only the last three lines. But as mentioned above maybe anyway not a good idea. :-/

Reply:
> For step3 it is only the last three lines

Yeah, that was me editing the message halfway, initially I only had the last step but then included everything and added the separators, without changing the text above.